modules/ROOT/pages/enabling-api-catalog.adoc

```diff
@@ -0,0 +1,33 @@
+= Enabling API Catalog
+
+Enable a connected Salesforce organization to access and sync Anypoint Platform APIs into the Salesforce API Catalog and expose them as invocable actions. This capability is enabled in Anypoint Platform Access Management by an Anypoint Platform organization administrator.  
+
+[[enable-disable-api-catalog]]
+== Enable or Disable API Catalog in Anypoint Platform
+
+include::partial$include-nav-steps-org-admin.adoc[]
+
+. Click *Salesforce*.
+. In *Capabilities Managed in Anypoint Platform*, click *Settings* for *Make MuleSoft Anypoint Platform APIs Available in Salesforce*.
+. In *Settings for Anypoint APIs in Salesforce*, click *Continue*.
+. In *Settings*, select the Anypoint Platform business groups that allow the Salesforce organization to import Anypoint Platform APIs into its API Catalog, and click *Save Settings*.
++
+[NOTE]
+Within the selected business groups, you can import only APIs that have specs published in Exchange with active instances into Salesforce.
```

Comment:
There are quite a prerequisites that have to be met for APIs to sync into API Catalog. I wonder if we could be a little less specific here and cross-reference the API Catalog "Before You Begin" (Name and link tbd) content in help.salesforce.com? Something like:
Within the selected business groups, you can sync only APIs from Exchange that have managed instances and meet the prerequisite conditions. For more information, see Salesforce Help. 

The alternative might be to duplicate all of the prereqs here, which becomes a maintenance issue. It's a bit of a moving target now.

Reply:
Also, import should be changed to sync throughout the content and the UI text.